### ISSUE TITLE
can this repo offer different model do inference time cost on the same gpu？

### ISSUE THREAD

**Issue by mlinxiang** [Cadene/pretrained-models.pytorch#134]
this repo is a very good accuracy rate benchmark reference for all model on imagenet classification task,  but when do real project deployment the model's inference time cost is also very important, so will this repo offer a benchmark test of this time cost, so can select the most balance model between the accuracy rate and inference time.

**Comment by Cadene** [comment]
@mlinxiang thanks for your interest. We are working on it :)

This study may help you; it used the pretrained-models.pytorch library:
https://arxiv.org/abs/1810.00736

**Comment by mlinxiang** [comment]
@Cadene perfect work, that's what i want, very thanks
